src/Cli/StartupJokeSpeechBubble.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buggregator\Trap\Cli;
+
+class StartupJokeSpeechBubble
+{
+    public const JOKES = [
+        'Why do programmers always mix up Halloween and Christmas? Because Oct 31 == Dec 25.',
+        'Two hard things in computer science: cache invalidation, naming things and stack overflow.',
+        'Depressive programming style through dump and die.',
+        'PHP was dead 84 years ago right?',
```

Comment:
@butschster are there any Buggregator related phrases and jokes in your baggage?